cms/apps/applications.json

```diff
@@ -398,6 +398,18 @@
       "twitter_URL": "https://x.com/astroport_fi",
       "medium_URL": "https://astroport.medium.com",
       "github_URL": "https://github.com/astroport-fi"
+    },
+    {
+      "title": "PampIT",
+      "subtitle": "Interchain Fair Token Launch Platform ",
+      "external_URL": "https://osmo.pampit.fun/",
+      "thumbnail_image_URL": "https://raw.githubusercontent.com/osmosis-labs/fe-content/main/cms/apps/images/pampit-cover.webp",
```

Comment:
This doesn't meet the requirements specified in the README (Notably, the aspect ratio). Please provide images that meet the requirements